src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -20,7 +20,7 @@ object ModelTypes {
 
 object ClusterStatus extends Enumeration {
   type ClusterStatus = Value
-  val Unknown, Creating = Value
+  val Unknown, Creating, Deleted, Deleting = Value
```

Comment:
nit: Deleting then Deleted (also you and vikram are gonna get merge conflicts)

Reply:
ok. yeah I can resolve them if he merges first.